core/lib/src/models/tables/study.dart

```diff
@@ -159,9 +159,8 @@ class Study extends SupabaseObjectFunctions<Study> {
   // ['id', 'title', 'description', 'published', 'icon_name', 'results', 'schedule']
   static Future<List<Study>> publishedPublicStudies() async {
     try {
-      return SupabaseQuery.extractSupabaseList<Study>(
-        await env.client.from(tableName).select(),
-      );
+      final response = await env.client.from(tableName).select().eq('participation', 'open') as List;
```

Comment:
Shouldn't this be checking for `registry_published` rather than `participation` now?
```suggestion
      final response = await env.client.from(tableName).select().eq('registry_published', true) as List;
```

Reply:
The method is used in the app. There we only want to show studies that can be accessed publicy i.e. participation=open.